openshift-ci: Ensure dracut is present on build root

The CI job on openshift-ci has failed for a while now. Currently we are using CentOS Stream 8 as build root and apparently dracut is missing on the build root image.

Failure example (https://storage.googleapis.com/origin-ci-test/logs/periodic-ci-kata-containers-kata-containers-main-e2e-tests/1500728468332089344/build-log.txt):
```
Creating initrd image based on the host OS using dracut
/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/dracut/add_libs.sh /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/dracut_overlay/usr/bin/kata-agent > /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/dracut/dracut.conf.d/15-extra-libs.conf
Agent appears to be linked statically, exiting
dracut --no-compress --conf /dev/null --confdir /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/dracut/dracut.conf.d --no-kernel --include /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/dracut_overlay / /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/kata-containers-initrd.img 
make: dracut: Command not found
make: *** [Makefile:143: /go/src/github.com/kata-containers/kata-containers/tools/osbuilder/kata-containers-initrd.img] Error 127
[install_kata_image.sh:53] ERROR: sudo -E BUILD_METHOD=dracut make -e initrd
[build_install.sh:148] ERROR: .ci/openshift-ci/../install_kata_image.sh
```

--- a/.ci/openshift-ci/build_install.sh
+++ b/.ci/openshift-ci/build_install.sh
@@ -145,6 +145,9 @@ fi
 # osbuilder's make define a VERSION variable which value might clash with
 # VERSION sourced from /etc/os-release.
 unset VERSION
+# It will be built a dracut-based rootfs so ensure dracut is installed in the
+# build environment.
+yum install -y dracut
 "${cidir}/install_kata_image.sh"
 
 "${cidir}/install_runtime.sh"

